src/main/java/Duke.java

```diff
@@ -26,6 +26,10 @@ public static void main(String[] args) {
                         int unmarkItem = sc.nextInt();
                         taskList.get(unmarkItem - 1).markUndone();
                         break;
+                    case "delete":
```

Comment:
Consider making inputs non-case sensitive! 
Try using 'String' 's equalsIgnoreCase() method compares two strings, ignoring lower case and upper case differences.